sagemathinc/cocalc
# Update to pre-privacy-shield

# Description
Update to our latest privacy policy without Privacy Shield.
<img width="863" alt="screen shot 2018-10-05 at 4 12 25 pm" src="https://user-images.githubusercontent.com/618575/46564042-da31c580-c8b9-11e8-917c-09d137862602.png">


# Testing Steps
1. Navigate to our privacy policy page ([base_url]/policies/privacy.html)
1. See that the date has been updated.

# Relevant Issues
None

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
